src/Pages/Home.tsx

```diff
@@ -1,17 +1,491 @@
-import NavMain from "../Components/NavMain";
 import HeroSection from "../Components/HeroSection";
-import LogoClouds from "../Components/LogoClouds";
+import AboutMe from "../Components/AboutMeSection";
+import ContactForm from "../Components/ContactForm";
+import MaxJGruberLogo from "../Assets/MaxJGruber_LOGO.png";
+import Title from "../Components/Title";
+import * as CVEn from "../Content/CVs/MaxJGruber_CV.pdf";
+import * as CVFr from "../Content/CVs/MaxJGruber_CV_FR.pdf";
```

Comment:
Aha! That's why you needed the module declaration. 

I don't understand how is JS even allowing this type of import...Imports are for .ts/.js files. I think you might just be lucky. 
Ideally here you should use the `fs` module (I don't know the equivalent for the frontend). 

Alternatively you can just check if it works in all the browsers and read about how and why it works (to be able to back it up if and interviewer asks you)